xcode/Subconscious/Shared/Models/StoryOnThisDay.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  StoryOnThisDay.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 30/9/2022.
+//
+
+import Foundation
+
+/// "on this day" story model
+struct StoryOnThisDay: Hashable, Identifiable, CustomStringConvertible {
```

Comment:
We could use StoryPrompt here and construct the prompt using `OnThisDayVariant` logic within the Geist service. That way we wouldn't need a new view and enum case.

Reply:
Yep, I was going to ask about this copy-pasting views pattern. Makes perfect sense to use `StoryPrompt` here since it's visually identical, will make the switch and delete the unused code.